lib/backend/syncersv1/updateprocessors/rules.go

```diff
@@ -154,6 +203,47 @@ func parseNamespaceSelector(s string) string {
 	return updated
 }
 
+// parseServiceAccounts takes a v2 service account selector and returns the appropriate v1 representation
+// by prefixing the keys with the `pcsa/` prefix. For example, `k == 'v'` becomes `pcsa/k == 'v'`.
```

Comment:
comment is slightly wrong, in that it also handles the entire match struct - including names.